File: examples/single-project/src/test/java/org/example/test/ExampleTest.java
package org.example.test;

import com.code_intelligence.jazzer.junit.FuzzTest;
import org.example.ExampleLib;

import static org.junit.jupiter.api.Assertions.assertTrue;

public class ExampleTest {

    @FuzzTest
    public void fuzzTest(byte[] data) {
        assertTrue(new ExampleLib().run());
    }
}

File: examples/single-project-scala/src/test/scala/org/example/test/ExampleTest.scala
package org.example.test

import com.code_intelligence.jazzer.junit.FuzzTest
import org.example.ExampleLib
import org.junit.jupiter.api.Assertions.assertTrue

class ExampleTest {

    @FuzzTest
    def testA(data: Array[Byte]) : Unit =
        assertTrue(new ExampleLib().run())

}